pluginsmanager/jeanam.code-workspace

```diff
@@ -0,0 +1,11 @@
+{
+	"folders": [
+		{
+			"path": "../../../jeanam"
```

Comment:
This patch is specific for your user

Reply:
I suppose that you need to create an entry into .gitignore for ignoring *.code-workspace